thirdparty/djvulibre/CMakeLists.txt

```diff
@@ -38,6 +38,13 @@ if(DEFINED ENV{ANDROID})
     set(PATCH_CMD3 "${KO_PATCH} ${CMAKE_CURRENT_SOURCE_DIR}/android.patch")
 endif()
 
+# Pick a few fixes from upstream
+set(PATCH_CMD4 "${KO_PATCH} ${CMAKE_CURRENT_SOURCE_DIR}/djvulibre-3.5.28-to-6a1e5.patch")
```

Comment:
Umm… why?

Reply:
I mean, why not just write `6a1e5ba1` some 5 lines down instead of some convoluted patch.

Reply:
Honestly? To make sure the diff applied cleanly? ;o)

Reply:
Oh, that :D. Yeah, generated the patch before checking that we used the git repo and not a tarball :D.